integrations/tensorflow/test/lit.cfg.py

```diff
@@ -44,21 +46,26 @@
     'imagenet_test_data.py',
 ]
 
+PYTHON_EXEC = sys.executable
+
+if PYTHON_EXEC is None:
+  PYTHON_EXEC = os.getenv("PYTHON")
+
 config.substitutions.extend([
-    ('%PYTHON', sys.executable),
+    ('%PYTHON', PYTHON_EXEC),
 ])
 
 # Add our local projects to the PYTHONPATH
 python_projects_dir = os.path.join(os.path.dirname(__file__), "..",
                                    "python_projects")
 test_src_dir = os.path.join(os.path.dirname(__file__), "python")
-llvm_config.with_environment("PYTHONPATH", [
+llvm_config.with_environment('PYTHONPATH', [
```

Comment:
Any reason for quote style intermingling? This file is somewhat inconsistent but seems double-quotish.

Reply:
Good point, made it all consistent double quote.